extensions/resource-deployment/src/ui/deployClusterWizard/pages/clusterSettingsPage.ts

```diff
@@ -224,51 +226,59 @@ export class ClusterSettingsPage extends WizardPageBase<DeployClusterWizard> {
 		const authModeDropdown = <azdata.DropDownComponent>this.inputComponents[VariableNames.AuthenticationMode_VariableName];
 		if (authModeDropdown) {
 			authModeDropdown.enabled = this.wizard.model.adAuthSupported;
-		}
-
-		this.wizard.wizardObject.registerNavigationValidator((pcInfo) => {
-			this.wizard.wizardObject.message = { text: '' };
-			if (pcInfo.newPage > pcInfo.lastPage) {
-				const messages: string[] = [];
-				const authMode = typeof authModeDropdown.value === 'string' ? authModeDropdown.value : authModeDropdown.value!.name;
-				const requiredFieldsFilled: boolean = !isInputBoxEmpty(getInputBoxComponent(VariableNames.ClusterName_VariableName, this.inputComponents))
-					&& !isInputBoxEmpty(getInputBoxComponent(VariableNames.AdminUserName_VariableName, this.inputComponents))
-					&& !isInputBoxEmpty(getInputBoxComponent(VariableNames.AdminPassword_VariableName, this.inputComponents))
-					&& !isInputBoxEmpty(getInputBoxComponent(ConfirmPasswordName, this.inputComponents))
-					&& (!(authMode === AuthenticationMode.ActiveDirectory) || (
-						!isInputBoxEmpty(getInputBoxComponent(VariableNames.OrganizationalUnitDistinguishedName_VariableName, this.inputComponents))
-						&& !isInputBoxEmpty(getInputBoxComponent(VariableNames.DomainControllerFQDNs_VariableName, this.inputComponents))
-						&& !isInputBoxEmpty(getInputBoxComponent(VariableNames.ClusterAdmins_VariableName, this.inputComponents))
-						&& !isInputBoxEmpty(getInputBoxComponent(VariableNames.ClusterUsers_VariableName, this.inputComponents))
-						&& !isInputBoxEmpty(getInputBoxComponent(VariableNames.DomainDNSIPAddresses_VariableName, this.inputComponents))
-						&& !isInputBoxEmpty(getInputBoxComponent(VariableNames.DomainDNSName_VariableName, this.inputComponents))));
-				if (!requiredFieldsFilled) {
-					messages.push(MissingRequiredInformationErrorMessage);
-				}
+			const adAuthSelected = (<azdata.CategoryValue>authModeDropdown.value).name === 'ad';
+			if (!this.wizard.model.adAuthSupported && adAuthSelected) {
+				this.formBuilder.removeFormItem(this.activeDirectorySection);
```

Comment:
>this.formBuilder.removeFormItem(this.activeDirectorySection); [](start = 4, length = 61)

Not sure how useful it'd be but I have a PR out to add the display property to components. Generally setting display to none for hidden items is better so you don't have to do a bunch of rebuilding of the dom as you add/remove items

Reply:
Looked at your PR, I can definitely leverage it. I will merge this PR and then use the new display property for all the pages if needed